src/api/routes/songs.ts

```diff
@@ -3,10 +3,52 @@ const data = require("../../../data/songdata.json");
 
 const router = express.Router();
 
+const fetchData = (key:string,value:string)=>{
```

Comment:
Please don't create a new function just for this. Also, use arrow functions to keep it all in one line.

Reply:
Done!